src/routes/signup/index.svelte

```diff
@@ -21,105 +25,195 @@
     }
   }
 
-  </script>
-  
-  <style>
-    .mzp-c-call-out {
-      background-color: transparent;
-      min-height: calc(100vh - 2.5rem);
-      padding: 0;
-    }
-  
-    .mzp-l-content {
-      padding-left: 0;
-      padding-right: 0;
-      /* this is Main + required padding to meet 120px */
-      padding-top: 80px;
-    }
-  
-    .mzp-c-call-out-title {
-      margin-bottom: 12px;
-    }
-    .how-it-works {
-      margin-top: 24px;
-      text-align: center;
-    }
-  
-    .mzp-c-field-label {
-      text-align: left;
+  const handleToggle = () => {
+    passwordVisible = !passwordVisible;
+    const type =
+      passwordEl.getAttribute("type") === "password" ? "text" : "password";
+    passwordEl.setAttribute("type", type);
+  };
+
+  const handleKeyup = (e) => {
+    const name = e.srcElement.name;
+    if (email && passwordEl) {
+      passwordEl.validity.valid && email.length > 0
+        ? (btnDisabled = false)
+        : (btnDisabled = true);
     }
-    .mzp-c-field-set {
-      width: max-content;
-      margin: auto;
+
+    if (password) {
+      if (name === "id_user_pw") {
+        password.length >= 8 ? (errorMsg = false) : (errorMsg = true);
```

Comment:
Do you think we're likely to change this in the future? I'd make it a `minPasswordLength` constant closer to the top of the file instead of hardcoding 8 here if so.

Reply:
Agreed!